tests/topotests/ldp-topo1/r1/ip_mpls_route.ref

```diff
@@ -1,5 +1,5 @@
 xx as to xx via inet 10.0.1.2 dev r1-eth0 proto xx
-xx as to xx via inet 10.0.1.2 dev r1-eth0 proto xx
```

Comment:
The commit message for this commit is oddly formated please fix.  It sure looks like some sort of cut-n-paste weirdness

Reply:
fixed